### PR TITLE
Operator: Unify naming of kubermatic resources

### PR DESCRIPTION
**Pull request by xrstf**
**What this PR does / why we need it**:
This PR takes the opportunity of the operator and tries to fix some longstanding naming inconsistencies:

* <del>The `-manager` suffix has been dropped, as all other controllers from other projects I've seen always just call it `acme-controller`, even if there are multiple controllers running inside.</del>
* The version suffixes have been dropped, as it's unrealistic to run two different APIs or UIs in the same namespace.
* The seed controller manager will not be called `kubermatic-controller-manager`, but `kubermatic-seed-controller-manager`.

I could also live with removing the `kubermatic-` prefix entirely, but having an "api" deployment can be confusing. On the other hand, if we keep the `kubermatic-` prefix, shouldn't our secrets et al. also be prefixed? /shrug

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

**Comment by alvaroaleman** [comment]
> The -manager suffix has been dropped, as all other controllers from other projects I've seen always just call it acme-controller, even if there are multiple controllers running inside.

Not true, its called `kubernetes-controller-_manager_`. Please keep the `manager` naming, as all of them run many controllers and not one.

**Comment by xrstf** [comment]
Fixed.

File: api/pkg/controller/operator-master/resources/kubermatic/api.go [hunk 1]
```diff
@@ -170,7 +169,7 @@ func APIDeploymentCreator(cfg *operatorv1alpha1.KubermaticConfiguration) reconci
 }
 
 func APIPDBCreator(cfg *operatorv1alpha1.KubermaticConfiguration) reconciling.NamedPodDisruptionBudgetCreatorGetter {
-	name := "kubermatic-api-v1"
```
Comment (alvaroaleman) [inline api/pkg/controller/operator-master/resources/kubermatic/api.go:173]: I bet the `const` linter will now complain :P

**Pull request merged**